### PR TITLE
#0: Update models/demos subdirectory code owners

### PR DESCRIPTION
**Pull request by bbradelTT**
The models/demos subdirectories had one owner. Add more owners.

### CODE REVIEW

File: CODEOWNERS
Comment (tt-rkim) [inline CODEOWNERS:132]: Please put @AleksKnezevic and @nsmithtt

File: CODEOWNERS
Comment (tt-rkim) [inline CODEOWNERS:136]: please replace Eyon with with @mtairum

File: CODEOWNERS
Comment (tt-rkim) [inline CODEOWNERS:137]: Please replace Eyon w/ @s-jovic

File: CODEOWNERS
Comment (tt-rkim) [inline CODEOWNERS:139]: Please delete Eyon

@cglagovichTT is there anyone else who can help with llama review in case you're on vacation?

File: CODEOWNERS
Comment (tt-rkim) [inline CODEOWNERS:142]: Please delete Saichand

**Comment by tt-rkim** [comment]
@bbradelTT has proposed having more owners of javelin models to ensure changes can go in if people are not available.

For example, @yieldthought is at a conference and @uaydonat is on vacation.

cc: @jvasilje @uaydonat

**Comment by eyonland** [comment]
Yeah, we need at least two people per slot.   I like the idea that model leads own these directories.

File: CODEOWNERS
Comment (s-jovic) [reply CODEOWNERS:135]: Can you add @pavlejosipovic @pavlepopovic and me here as well?

File: CODEOWNERS
Comment (bbradelTT) [reply CODEOWNERS]: Done.

File: CODEOWNERS
Comment (bbradelTT) [reply CODEOWNERS]: Done

File: CODEOWNERS
Comment (bbradelTT) [reply CODEOWNERS]: Done

File: CODEOWNERS
Comment (bbradelTT) [reply CODEOWNERS]: Done

File: CODEOWNERS
Comment (bbradelTT) [reply CODEOWNERS]: Done

File: CODEOWNERS
Comment (bbradelTT) [reply CODEOWNERS]: Done

File: CODEOWNERS
Comment (bbradelTT) [reply CODEOWNERS]: Done

File: CODEOWNERS
Comment (bbradelTT) [reply CODEOWNERS]: Done

**Comment by bbradelTT** [comment]
@tt-rkim the owners of the non-model lines are:
```
SeanNijjar
aliuTT
mo-tenstorrent
pgkeller
rtawfik01
tapspatel
tt-aho
tt-nshanker
tt-rkim
ubcheema
```
It would be good if you updated those lines in a subsequent PR since you have a better idea about who to add as secondaries for those files.

File: CODEOWNERS
Comment (AleksKnezevic) [inline CODEOWNERS:131]: Can @nsmithtt and I be added to the wormhole demos as well?

Comment (bbradelTT) [reply CODEOWNERS:131]: Just `models/demos/wormhole`?
Or  `models/demos/wormhole`, `models/demos/wormhole/falcon7b`, and `models/demos/wormhole/mistral7b`?

Comment (AleksKnezevic) [reply CODEOWNERS:131]: I need `stable_diffusion`, which will be moved into `models/demos/wormhole`

Comment (bbradelTT) [reply CODEOWNERS]: Ok. Done.

File: CODEOWNERS
Comment (bbradelTT) [reply CODEOWNERS]: Done

**Review by tt-rkim — APPROVED** [review]

**Comment by tt-rkim** [comment]
@bbradelTT By the way in case you weren't aware, these changes will also force owners who were originally alone to now have to get approval from others. Co-owners cannot change files at will. Just in case you thought that was the case

**Comment by bbradelTT** [comment]
> @bbradelTT By the way in case you weren't aware, these changes will also force owners who were originally alone to now have to get approval from others. Co-owners cannot change files at will. Just in case you thought that was the case

@tt-rkim That's frustrating. Unless there are three people per line having only two people would cause the same problem as before, except for people that will most likely make changes. I guess there are no plans to change that, right?

**Comment by uaydonat** [comment]
@mtairum should be added to mixtral.

@xuncaiTT should be added to llama*

yes, having multiple approvers makes sense if people are away to keep things moving. However, unless the main owners (first in the list) are away (or submitting code themselves), they should be in charge of the approvals.

**Review by s-jovic — APPROVED** [review]

**Comment by bbradelTT** [comment]
@tt-rkim @xuncaiTT

I updated the file based on the newest comment and I'm getting the following error:
`Unknown owner on line 161: make sure @xuncaiTT exists and has write access to the repository`

@xuncaiTT does exist and seems to have commits to the repository.

Any idea what's going on?

**Comment by tt-rkim** [comment]
@bbradelTT Pls use @caixunshiren to specify Jack instead of @xuncaiTT 
He currently uses the former account.

@uaydonat FYI

**Comment by uaydonat** [comment]
@djordje-tt should also be in models/demos/wormhole/falcon7b

**Pull request merged**
